gems/sorbet-runtime/lib/types/sig.rb

```diff
@@ -7,18 +7,32 @@ module T::Sig
   module WithoutRuntime
     # At runtime, does nothing, but statically it is treated exactly the same
     # as T::Sig#sig. Only use it in cases where you can't use T::Sig#sig.
-    def self.sig(&blk); end # rubocop:disable PrisonGuard/BanBuiltinMethodOverride
+    def self.sig(&blk) # rubocop:disable PrisonGuard/BanBuiltinMethodOverride
+      T::Private::Methods::DeclarationBlock.new(T.unsafe(nil), T.unsafe(nil), T.unsafe(nil))
+    end
```

Comment:
I don't like this.

Reply:
I don't really understand what this is for. Could you just use default arguments? Maybe with keyword arguments?